setup.cfg

```diff
@@ -0,0 +1,8 @@
+[flake8]
+max-line-length = 110
+ignore = E133, E226, E228, N802, N803, N806
```

Comment:
Does telescope and site have the same ignores as DM? In theory T&S could be different.

Reply:
T&S has no standard so I adopted DM standards.